.firebaserc

```diff
@@ -0,0 +1,5 @@
+{
+  "projects": {
+    "default": "atlassian-connect"
```

Comment:
Heet het project `atlassian-connect`? Ik zou dit even fixen naar bijv: `ecfv-dev` die kon ik nu vinden in firebase